src/shared/components/ActionBar/ActionBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import {
+  StyledActionBarContainer,
+  StyledInfoContainer,
+  StyledPrimaryText,
+  StyledSecondaryText,
+  StyledDetailsTextContainer,
+  StyledButtonsContainer,
+  StyledSecondaryButton,
+} from './ActionBar.style'
+import { Button, Icon } from '@/shared/components'
+import type { IconType } from '../Icon'
+
+export type ActionBarProps = {
+  primaryText?: string
+  secondaryText?: string
+  primaryButtonText?: string
+  detailsText?: string
+  detailsTextIcon: IconType
+  secondaryButtonText?: string
+  secondaryButtonIcon?: IconType
+  onClickPrimaryButton?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+  onClickSecondaryButton?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+}
+
+const ActionBar: React.FC<ActionBarProps> = ({
+  primaryText,
+  secondaryText,
+  primaryButtonText,
+  secondaryButtonText,
+  detailsText,
+  detailsTextIcon,
+  secondaryButtonIcon,
+  onClickPrimaryButton,
```

Comment:
So far we've only used `on...Click`, for example `onEditClick`, `onChannelClick`, let's stay consistent with that naming